plugins/main/public/components/endpoints-summary/table/agents-table.tsx

```diff
@@ -0,0 +1,483 @@
+
+    return (
+      <EuiFlexGroup gutterSize='xs'>
+        <EuiFlexItem grow={false}>
+          <i
+            className={`fa fa-${icon} AgentsTable__soBadge AgentsTable__soBadge--${icon}`}
+            aria-hidden='true'
+          ></i>
+        </EuiFlexItem>{' '}
+        <EuiFlexItem>{os_name.trim() || '-'}</EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  };
+
+  // Columns with the property truncateText: true won't wrap the text
+  // This is added to prevent the wrap because of the table-layout: auto
+  const defaultColumns = [
+    {
+      field: 'id',
+      name: 'ID',
+      sortable: true,
+      show: true,
+      searchable: true,
+    },
+    {
+      field: 'name',
+      name: 'Name',
+      sortable: true,
+      show: true,
+      searchable: true,
+    },
+    {
+      field: 'ip',
+      name: 'IP address',
+      sortable: true,
+      show: true,
+      searchable: true,
+    },
+    {
+      field: 'group',
+      name: 'Group(s)',
+      sortable: true,
+      show: true,
+      render: groups => renderGroups(groups),
+      searchable: true,
+    },
+    {
+      field: 'os.name,os.version',
+      composeField: ['os.name', 'os.version'],
+      name: 'Operating system',
+      sortable: true,
+      show: true,
+      render: (field, agentData) => addIconPlatformRender(agentData),
+      searchable: true,
+    },
+    {
+      field: 'node_name',
+      name: 'Cluster node',
+      sortable: true,
+      show: true,
+      searchable: true,
+    },
+    {
+      field: 'version',
+      name: 'Version',
+      sortable: true,
+      show: true,
+      searchable: true,
+    },
+    {
+      field: 'dateAdd',
+      name: (
+        <span>
+          Registration date{' '}
+          <EuiIconTip
+            content='This is not searchable through a search term.'
+            size='s'
+            color='subdued'
+            type='alert'
+          />
+        </span>
+      ),
+      render: dateAdd => formatUIDate(dateAdd),
+      sortable: true,
+      show: false,
+      searchable: false,
+    },
+    {
+      field: 'lastKeepAlive',
+      name: (
+        <span>
+          Last keep alive{' '}
+          <EuiIconTip
+            content='This is not searchable through a search term.'
+            size='s'
+            color='subdued'
+            type='alert'
+          />
+        </span>
+      ),
+      render: lastKeepAlive => formatUIDate(lastKeepAlive),
+      sortable: true,
+      show: false,
+      searchable: false,
+    },
+    {
+      field: 'status',
+      name: 'Status',
+      truncateText: true,
+      sortable: true,
+      show: true,
+      render: (status, agent) => <AgentStatus status={status} agent={agent} />,
+    },
+    {
+      field: 'group_config_status',
+      name: 'Synced',
+      sortable: true,
+      show: false,
+      render: synced => <AgentSynced synced={synced} />,
+      searchable: true,
+    },
+    {
+      name: 'Actions',
```

Comment:
In order to hide/show the actions column, the `field: 'actions'` must be specified.

Reply:
Done